assets/js/base/components/checkbox-list/index.js

```diff
@@ -97,21 +97,23 @@ const CheckboxList = ( {
 		return (
 			<Fragment>
 				{ options.map( ( option, index ) => (
-					<Fragment key={ option.key }>
+					<Fragment key={ option.value }>
```

Comment:
I'm curious why this change?  I know you changed the expected shape from `option.key` to `option.value` (so `option.key` no longer exists`) Is this just to make it more consistent (being it is the _value_ for the option)?

Reply:
There is no reason in fact, I modified it initially because the Downshift example was using `value` instead of `key` and to get started it was easier to modify the data structure of `options`.

Later, I didn't undo the change because both `key` and `value` sound good to me. But I don't mind undoing it if you think `key` was better.